### PR TITLE
Docs OSS Quickstart guide revision

### LINKED ISSUE #48780: Simplify and improve the OSS Quickstart guide
### Page URL

https://docs.airbyte.com/using-airbyte/getting-started/oss-quickstart

### Description

For many, the open source Quickstart guide is one of the first interactions they'll have with Airbyte. However, the current version of this guide is not "quick" in nature, and it contains a great deal of superfluous or more advanced information. This has helped make the topic one of our most downvoted documentation topics. For a fuller analysis of deployment docs issues, see [This Google Doc](https://docs.google.com/document/d/10rJo6Miv-nZdWDnpLti9cyUoJXltKSVQ5-NF5Vc9stQ/edit?tab=t.0).

Fixing this topic should be our first priority on the road to more user-friendly, user-oriented, goal-based deployment documentation. This task will rewrite the Quickstart as an extremely focused, straightforward explanation of how to set up a local instance of Airbyte. All other information will be preserved for posterity or moved to the Deployment section for future revision.

### PR DESCRIPTION
**Pull request by ian-at-airbyte**
## What

A new, Open Source Quickstart guide that is, well, quick.

## How

Rewrites the OSS Quickstart guide. This is a more focused, minimum-steps approach that omits any information that isn't strictly necessary to get Airbyte running. That extra additional information is being temporarily preserved in the deployment guide until that is redone later. It's not meant to exist in its current form indefinitely.

## Review guide

1. /using-airbyte/getting-started/oss-quickstart - this file has been heavily edited and should be reviewed end-to-end.
2. docusaurus/docusaurus.config.js - add syntax highlighting for terminal/bash commands
3. docusaurus/src/css/custom.css - style changes to improve the legibility of highlighted lines of code (highlighted lines were almost unreadable in light mode).

The other changes are not really important. They're just to avoid losing anything someone might need.

## User Impact

The tighter focus around just getting Airbyte running is intended to make the first-touch easier and get people using Airbyte. This should help reduce the many downvotes this topic has received in the past, but probably won't fully achieve this goal in isolation. Later planned changes to the full deployment guide are critical to improving how this content is received.

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: docusaurus/docusaurus.config.js [hunk 1]
```diff
@@ -189,6 +189,7 @@ const config = {
       prism: {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
+        additionalLanguages:['bash'],
```
Comment (ian-at-airbyte) [inline docusaurus/docusaurus.config.js:192]: Enables syntax highlighting for terminal commands.

File: docusaurus/src/css/custom.css
Comment (ian-at-airbyte) [inline docusaurus/src/css/custom.css:1]: Makes highlighted lines in code samples legible.

File: docs/using-airbyte/getting-started/oss-quickstart.md [hunk 2]
```diff
@@ -7,361 +7,276 @@ import TabItem from "@theme/TabItem";
...
+    You can use that password to log into Airbyte, but you probably want to use your own email and password.
 
-1. Install the docker engine:
+3. Set an email.
 
-```shell
-sudo yum install -y docker
-```
+    ```bash
+    abctl local credentials --email your.name@example.com
```
Comment (bgroff) [inline docs/using-airbyte/getting-started/oss-quickstart.md:222]: While this still works, I think you actually set your email during the initial setup now, so hopefully this is less necessary. @ian-at-airbyte would love to get your opinion on the work flow there. Lets go over it next week during our sync.

File: docs/deploying-airbyte/abctl-ec2.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+   :::tip
+   By default, `abctl` only configures an ingress rule for the host `localhost`. In order to ensure that Airbyte can be accessed outside of the EC2 instance, you will need to specify the `--host` flag to the `local install` command, providing the FQDN of the host which is hosting Airbyte. For example, `abctl local install --host airbyte.company.example`.
+   :::
+
+By default, `abctl` will listen on port 8000. If port 8000 is already in used or you require a different port, you can specify this by passing the `--port` flag to the `local install` command. For example, `abctl local install --port 6598`
+
+Ensure the security group configured for the EC2 Instance allows traffic in on the port (8000 by default, or whatever port was passed to `--port`) that you deploy Airbyte on. See the [Control traffic to your AWS resources using security groups](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-security-groups.html) documentation for more information.
+
+```shell
+abctl local install --host [HOSTNAME]
```
Comment (bgroff) [inline docs/deploying-airbyte/abctl-ec2.md:53]: We should probably say something about how the host is no longer required and in which cases you might want to set a host (it is a security feature).

**Review by bgroff — APPROVED** [review]

File: docs/using-airbyte/getting-started/oss-quickstart.md [hunk 4]
```diff
@@ -7,361 +7,276 @@ import TabItem from "@theme/TabItem";
...
-  Password: random_password
-  Client-Id: 03ef466c-5558-4ca5-856b-4960ba7c161b
-  Client-Secret: m2UjnDO4iyBQ3IsRiy5GG3LaZWP6xs9I
-```
-
-Use the value in the password field to authenticate to your new Airbyte instance.
+1. Verify your processor architecture.
 
-You can set your email and password with the `credentials` command using `abctl`. To set your email you can run:
+    1. Press <kbd><svg xmlns="http://www.w3.org/2000/svg" height="10" width="8.75" viewBox="0 0 448 512"><path d="M0 93.7l183.6-25.3v177.4H0V93.7zm0 324.6l183.6 25.3V268.4H0v149.9zm203.8 28L448 480V268.4H203.8v177.9zm0-380.6v180.1H448V32L203.8 65.7z"/></svg></kbd> + <kbd>I</kbd>.
```
[the remark this answers is not in the document]
Comment (ian-at-airbyte) [reply docs/using-airbyte/getting-started/oss-quickstart.md]: Good call! I've converted these to proper FontAwesome icons, which should be fully accessible and visible in both colors modes.

File: docs/using-airbyte/getting-started/oss-quickstart.md (hunk 2 above)
Comment (ian-at-airbyte) [reply docs/using-airbyte/getting-started/oss-quickstart.md]: I've updated the steps to reflect this. I think the UI-based way makes sense, and it's redundant to run the command now.

**Pull request merged**